config/params_template.env

```diff
@@ -65,3 +65,8 @@ export manual_calb_toggle="True"
 export ncores_gw=1 # mpi number of cores for gagewatershed
 export ncores_fd=1 # mpi number of cores for flow directions
 export memfree=0G # min free memory required to start a new job or keep youngest job alive
+
+#### Healed HAND ####
+# Removes Hydro-conditioning Artifacts
```

Comment:
Since your code is not in post-processing, I would like to move your section up above the others that are in post-processing. The proper place should be after the levees and before the bathymetry adjustment on line `33`.

Also, can you remove the last commented line, but keep the `true=on` part? The comments should read like this:
```
#### Healed HAND ####
# Removes hydro-conditioning artifacts (true=on; false=off)
```

Reply:
The suggested changed were made.